app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user
+  before_action :load_user, only: :show
+  
+  def show; end
+
+  private
+
+  def logged_in_user
+    return if logged_in?
+
+    store_location
+    flash[:danger] = t "message.alert_login"
+    redirect_to login_url
+  end
+
+  def load_user
+    @user = User.find_by id: params[:id]
+    return if @user
```

Comment:
Line 18 va 19 Viết gọn lại
```suggestion
    return if @user = User.find_by(id: params[:id])
```

Reply:
đỉnh quá anh ơi